app/reactions/mutations/createReaction.ts

```diff
@@ -17,8 +17,11 @@ async function createReaction({ data }: CreateReactionInput, ctx: Ctx) {
   })
   if (existingReaction) return existingReaction
 
+  // Add the user connection to data
+  data.user = { connect: { id: ctx.session.userId } }
```

Comment:
Why this instead of spreading? Just curious...

Reply:
I don't know exactly why, but when the spreading is used, the ide throws a types error. I suspect that something new about blitz + spreading `data` onto a non typed object makes typescript confused, but I got surprised of this too.
![image](https://user-images.githubusercontent.com/40028772/106305638-b093c980-623b-11eb-8224-7c859e4e31af.png)

Reply:
Spookie 👻